### DISCUSSION TITLE
Pair Extraordinaire badge

### DISCUSSION THREAD

**Discussion by rimiit** [community/community#48613] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

How to get the Pair Extraordinaire badge?

**Comment by samanta-amit** [comment, marked as answer]
You need to [Coauthored](https://docs.github.com/pull-requests/committing-changes-to-your-project/creating-and-editing-commits/creating-a-commit-with-multiple-authors) in a merged pull request.
